src/CodeGen_Hexagon.cpp

```diff
@@ -283,8 +283,6 @@ class SloppyUnpredicateLoadsAndStores : public IRMutator {
 
             return Call::make(op->type, Call::if_then_else,
                               {condition, load, make_zero(op->type)}, Call::Intrinsic);
-
-            return load;
```

Comment:
Ouch. I wonder if this was a bad merge. Probably worth turning on this warning explicitly.

Reply:
Yow. I'm genuinely surprised this is merely a warning.